charts/ff-proxy/values.yaml

```diff
@@ -2,34 +2,63 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-# The ProxyKey you want to configure your Proxy to use
-proxyKey:
-
-# Use existing Secret which stores PROXY_KEY key instead of creating a new one. The value should be set with the `PROXY_KEY` key inside the secret.
-## If set, this parameter takes precedence over "proxyKey".
-existingProxyKey:
-
-# Used by the Proxy to sign the JWT tokens it creates and returns to SDKs when they authenticate with the Proxy. The Proxy then checks that the token provided in any subsequent reqeusts by SDKs has been signed with this secret to ensure auth tokens can't be spoofed.
-authSecret:
-
-# Use existing Secret which stores AUTH_SECRET key instead of creating a new one. The value should be set with the `AUTH_SECRET` key inside the secret.
-## If set, this parameter takes precedence over "authSecret".
-existingAuthSecret:
-
-# The host and port of the redis server you want your Proxy to connect to
-redis:
-  address:
-  db: 0
-  password:
-  # Use existing Secret which stores REDIS_PASSWORD key instead of creating a new one. The value should be set with the `REDIS_PASSWORD` key inside the secret.
-  ## If set, this parameter takes precedence over "redis.password".
-  existingPassword:
+global:
+  loadbalancerURL: ""
+  database:
+    redis:
+      installed: true
+      protocol: "redis"
+      # --  provide default values if redis.installed is set to false
+      hosts:
+        - redis:6379
+      ssl:
+        enabled: false
+        secret: ""
+        caFileKey: ""
+        trustStoreKey: ""
+        trustStorePasswordKey: ""
+      secretName: ""
+      userKey: ""
+      passwordKey: ""
+      extraArgs: ""
+      secrets:
+        kubernetesSecrets:
+          - secretName: ""
+            keys:
+              REDIS_USERNAME: ""
+              REDIS_PASSWORD: ""
+        secretManagement:
+          externalSecretsOperator:
+            - secretStore:
+                name: ""
+                kind: ""
+              remoteKeys:
+                REDIS_USERNAME:
+                  name: ""
+                  property: ""
+                REDIS_PASSWORD:
+                  name: ""
+                  property: ""
+  waitForInitContainer:
+    image:
+      registry: docker.io
+      repository: harness/helm-init-container
+      pullPolicy: Always
+      tag: "latest"
+      digest: ""
+      imagePullSecrets: []
+  monitoring:
+    enabled: true
+    port: 8889
```

Comment:
Remove port and path from the global and set it in the local?

Reply:
yes, I did it, ill remove this usage too